packer/provisioner.go

```diff
@@ -145,15 +145,61 @@ func (p *PausedProvisioner) Prepare(raws ...interface{}) error {
 
 func (p *PausedProvisioner) Provision(ctx context.Context, ui packersdk.Ui, comm packersdk.Communicator, generatedData map[string]interface{}) error {
 
-	// Use a select to determine if we get cancelled during the wait
 	ui.Say(fmt.Sprintf("Pausing %s before the next provisioner...", p.PauseBefore))
+
+	var err error
+	switch ui.(type) {
+	case *MachineReadableUi:
+		err = p.pausingNoUpdate(ctx)
```

Comment:
I tested but I don't know why it doesn't recognize as the MachineReadableUI here 🤔  Did it work for you?

Reply:
I thought this was handled by [this code](https://github.com/teddylear/packer/blob/feature/provisionor-pause_before-countdown-2/packer/provisioner.go#L152-L153) and [this corresponding test](https://github.com/teddylear/packer/blob/feature/provisionor-pause_before-countdown-2/packer/provisioner_test.go#L180-L214). Please let me know if there's anything I'm missing or anything else I should be testing for.